gui/src/locales/en.json

```diff
@@ -125,7 +125,8 @@
         "email": "Email",
         "login": "Login",
         "logout": "Logout",
-        "continue": "Continue"
+        "continue": "Continue",
+        "NA": "NA"
```

Comment:
Please use snake_case for keys and need to follow this convention throughout.
(All lowercase and spaces to be separated by _)

Reply:
Updated to kabab-case, as stated.

Reply:
Please use snake_case (All lowercase and spaces to be separated by _) not kebab-case

Reply:
Updated NA to not-available - "snake-case".